pkg/api/helper/helpers.go

```diff
@@ -107,10 +107,13 @@ func IsResourceQuotaScopeValidForResource(scope api.ResourceQuotaScope, resource
 var standardContainerResources = sets.NewString(
 	string(api.ResourceCPU),
 	string(api.ResourceMemory),
+	string(api.ResourceStorageOverlay),
```

Comment:
@jingxu97 You need to remove these resources once you finalize the resource APIs for storage.